src/screens/groups/components/recipients-list/index.ts

```diff
@@ -1 +1,2 @@
 export { default as RecipientsList } from './recipients-list'
+export { default as BatchRecipientsList } from './batch-recipients-list'
```

Comment:
Frankly speaking, this component should be in separate folder, because there is no common behaviour between two different lists. And `batch-transaction-dialog` also.

Reply:
shall i create two directories ? 1] batch-recipients-list 2] batch-transaction-dialog and then keeping their respective file in them, WDYT